Mathlib/Topology/Category/CompHaus/ExplicitLimits.lean

```diff
@@ -36,10 +36,9 @@ pairs `(x,y)` such that `f x = g y`, with the topology induced by the product.
 -/
```

Comment:
Ah, I see, this file already existed. I think this probably should just be `Limits.lean`.